src/client/scripts/game/gui/stats.js

```diff
@@ -16,111 +16,125 @@ import config from '../config.js';
+let visibilityWeight = 0;
+
+/**
+ * Temporarily displays the move number in the corner of the screen.
+ * @param {number} [durationSecs] The duration to show the move number. Default: 2.5
+ */
+function showMoves(durationSecs = 2.5) {
+    if (config.VIDEO_MODE) return;
+
+    visibilityWeight++;
+
+    setTextContentOfMoves();
+    setTimeout(hideMoves, durationSecs * 1000);
+
+    if (visibilityWeight === 1) style.revealElement(elementStatusMoves);
+}
+
+function hideMoves() {
+    visibilityWeight--;
+    if (visibilityWeight === 0) style.hideElement(elementStatusMoves);
+}
+
+function setTextContentOfMoves( ) {
+
+    const currentPly = game.getGamefile().moveIndex + 1;
+    const totalPlyCount = movesscript.getPlyCount(game.getGamefile().moves);
+
+    elementStatusMoves.textContent = `${translations.move_counter} ${currentPly}/${totalPlyCount}`;
+}
+
+function updateStatsCSS() {
+    element_Statuses.style = `top: ${camera.getPIXEL_HEIGHT_OF_TOP_NAV()}px`;
+}
+
+function showPiecesMesh() {
+    if (config.VIDEO_MODE) return;
+    style.revealElement(elementStatusPiecesMesh);
+}
+
+function updatePiecesMesh(percent) {
+    const percentString = math.decimalToPercent(percent);
+    elementStatusPiecesMesh.textContent = `${translations.constructing_mesh} ${percentString}`;
+}
+
+function hidePiecesMesh() {
+    style.hideElement(elementStatusPiecesMesh);
+}
+
+function showFPS() {
+    if (config.VIDEO_MODE) return;
+    style.revealElement(elementStatusFPS);
+}
+
+function hideFPS() {
+    style.hideElement(elementStatusFPS);
+}
+
+function updateFPS(fps) {
+    if (!options.isFPSOn()) return;
+    const truncated = fps | 0; // Bitwise operation that quickly rounds towards zero
+    elementStatusFPS.textContent = `FPS: ${truncated}`;
+}
+
+function showRotateMesh() {
+    if (config.VIDEO_MODE) return;
+    style.revealElement(elementStatusRotateMesh);
+}
+
+function updateRotateMesh(percent) {
+    const percentString = math.decimalToPercent(percent);
+    elementStatusRotateMesh.textContent = `${translations.rotating_mesh} ${percentString}`;
+}
+
+function hideRotateMesh() {
+    style.hideElement(elementStatusRotateMesh);
+}
+
+// NO LONGER USED. These were for the aynchronious checkmate algorithm.
+// showMoveLooking() {
+//     if (config.VIDEO_MODE) return;
+//     style.revealElement(elementStatusMoveLooking);
+// },
+// updateMoveLooking(percent) {
+//     const percentString = math.decimalToPercent(percent);
+//     showMoveLooking();
+//     elementStatusMoveLooking.textContent = `Looking for moves... ${percentString}`;
+// },
+
+function hideMoveLooking() {
+    style.hideElement(elementStatusMoveLooking);
+}
+
+export default {
+    showMoves,
+    hideMoves,
+    setTextContentOfMoves,
+    updateStatsCSS,
+    showPiecesMesh,
+    updatePiecesMesh,
+    hidePiecesMesh,
+    showFPS,
+    hideFPS,
+    updateFPS,
+    showRotateMesh,
+    updateRotateMesh,
+    hideRotateMesh,
+    hideMoveLooking
```

Comment:
hideMoves() is not called externally, neither:
setTextContentOfMoves,

and hideMoveLooking() is depricated, may as well delete.